postgresql-containers/build/pgbackrest/Dockerfile

```diff
@@ -98,7 +98,7 @@ RUN set -ex; \
     rpmkeys --checksig /tmp/perl-DBI.rpm /tmp/perl-XML-Parser.rpm /tmp/perl-libxml-perl.rpm /tmp/perl-DBD-Pg.rpm; \
     rpm -i /tmp/perl-DBI.rpm /tmp/perl-XML-Parser.rpm /tmp/perl-libxml-perl.rpm /tmp/perl-DBD-Pg.rpm; \
     rm -rf /tmp/perl-DBI.rpm /tmp/perl-XML-Parser.rpm /tmp/perl-libxml-perl.rpm /tmp/perl-DBD-Pg.rpm; \
-    microdnf -y install  \
+    microdnf -y --enablerepo=epel install  \
```

Comment:
strange, percona-pgbackrest for PG15 has dependency to libssh2.so.1, percona-pgbackrest for PG16 does not